autoprotocol/container_type.py

```diff
@@ -1238,6 +1238,84 @@ def row_count(self):
     cat_no="136101",
 )
 
+#:
+FLAT384BLACKUBOTTOM = ContainerType(
+    name="384-well Black Clear Round Bottom Ultra-low Attachment Microplate",
+    shortname="384-ubottom-black-clear-tc",
+    is_tube=False,
+    well_coating=None,
+    sterile=True,
+    well_count=384,
+    col_count=24,
+    well_depth_mm=12.55,
```

Comment:
make well_depth_mm a Unit:

well_depth_mm = Unit(12.55, "millimeter"),

Reply:
The signature asks a float type. And most of the containers in this module passes a float, only two don't follow the signature.